src/Repository/ServerManager/GameListRepository.php

```diff
@@ -116,29 +121,35 @@ private function amendSessionList(array &$sessionList): void
             $sessionList[$key] = $session;
         }
     }
+    public function defaultDenormalizeContext(): array
+    {
```

Comment:
member names via reflection ? 
array in static cachen?

Reply:
will have to tackle this some other time - need to move on to other priority tasks